krux/cli.py

```diff
@@ -96,6 +96,8 @@ def __init__(self, name, parser=None, logger=None, lockfile=False):
 
         :keyword parser: The CLI parser. Defaults to
         :py:func:`cli.get_parser <krux.cli.get_parser>`
+
+        :keyword cli_args: The cli arguments. when it = '', no any argument will be passed to CLI application.
```

Comment:
Actually at here is a kind of hack. If cli_args = None, no any change will applied by this patch, and the application will use sys.argv arguments. And if cli_args = '', the application will not use any argument. 

Thanks @anishbhuta, I will change to use this comment instead: "List of arguments to use as command-line arguments instead of sys.argv (for tests)"